packages/devtools_app/lib/src/app.dart

```diff
@@ -440,6 +440,11 @@ class SettingsDialog extends StatelessWidget {
             listenable: preferences.darkModeTheme,
             toggle: preferences.toggleDarkModeTheme,
           ),
+          _buildOption(
```

Comment:
It would be nice to make both these checkboxes use NotifierCheckbox which is a little cleaner as there is not a separate toggle method and listenable.

Reply:
I agree we should find a way to merge the functionality here. One thing this has that NotifierCheckbox does not is that the checkbox label is also clickable. Additionally, some of these options have toggle callbacks that do more than just update the notifier, so replacement won't be trivial. I will add a todo.